### PR TITLE
[6.0.0]Add script for AA enable

### PR DESCRIPTION
**Pull request by Thumimku**
## Purpose
$sub
aadoc

Issue : https://github.com/wso2/product-is/issues/14135

### CODE REVIEW

File: en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md [hunk 1]
```diff
@@ -17,6 +17,24 @@ This tutorial guides you through setting up a sample application to try
...
+    You need to run **adaptive.sh** (**adaptive.bat** for Windows) in 
+    `          [IS-HOME]/bin         `
+
+    To enable adaptive authentication please proceed following instructions.
+
+    1.   Stop the server if running
+    2.   Run adaptive.sh (adaptive.bat for Windows) (eg: `sh adaptive.sh`)
+    3.   Restart the server
+ 
+    To disable adaptive authentication please proceed following instructions.
```
Comment (piraveena) [inline en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md:32]: shall we add this at the end of the doc?

Comment (Thumimku) [reply en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md:32]: may I know the reason why? ... if we add at the end of it, there may 2 issues

1. Developer may miss it
2. Developers need to restart after he/she create app and configure

Comment (piraveena) [reply en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md:32]: The title says as `Before you begin`

File: en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md [hunk 2]
```diff
@@ -17,6 +17,24 @@ This tutorial guides you through setting up a sample application to try
 out adaptive authentication and configuring it as a service provider in
 WSO2 IS.
 
+!!! Warning "Before you proceed"
+
+    Adaptive Authentication is disabled by default.
+    You need to run **adaptive.sh** (**adaptive.bat** for Windows) in
```
Comment (divyaamunugama) [inline en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md:23]: The name of the script in the latest pack is **enableadaptive.sh**

Comment (Thumimku) [reply en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md:23]: ACK

**Pull request closed without merging**